app/src/main/java/com/mybus/model/SearchType.java

```diff
@@ -0,0 +1,14 @@
+package com.mybus.model;
+
+/**
+ * Created by Julian Gonzalez <jgonzalez@devspark.com>
+ */
+public final class SearchType {
```

Comment:
Maybe you can use an enum and the ordinal method to get the int position of each value.

```java
public enum SearchType {
	ORIGIN, DESTINATION, FAVORITE
}

int value = SearchType.ORIGIN.ordinal(); // value = 0
int value = SearchType.DESTINATION.ordinal(); // value = 1
int value = SearchType.FAVORITE.ordinal(); // value = 2
```